src/KeyringController.ts

```diff
@@ -36,16 +43,16 @@ class KeyringController extends EventEmitter {
 
   public memStore: ObservableStore<KeyringControllerState>;
 
-  public encryptor: any;
-
   public keyrings: Keyring<Json>[];
 
-  public cacheEncryptionKey: boolean;
-
   public unsupportedKeyrings: SerializedKeyring[];
 
   public password?: string;
 
+  #encryptor: GenericEncryptor | KeyEncryptor;
+
+  #cacheEncryptionKey: boolean;
```

Comment:
These has been made unaccessible, so that we can add a getter and a setter for each of them, and add validations on setters

Reply:
Have you considered making this fully private? We don't have any reason to access the encyrptor at runtime, or to change it at runtime. The constructor parameter seems adequate to me, and reduces the API surface area

Reply:
Good point, I agree! Refactored in aff13b04678d025059c6e3ccf3c6a1ca989e358f
This change also required some tests refactoring